CHANGELOG.md

```diff
@@ -136,6 +138,38 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixes
 - Infinite recursion in mixins middleware ([b5a4317](https://github.com/nunomaduro/larastan/commit/b5a4317ef7c19b9008e4efff7ef50d2649b00151))
 
-## [0.1.0] - 2018-07-17
+## 0.1.0 - 2018-07-17
```

Comment:
Can you put the brackets again?

Reply:
Well there is no link to 0.1.0 ?
Followed the same convention as [Keep a Changlog](https://github.com/olivierlacan/keep-a-changelog/blob/master/CHANGELOG.md)

If you want them back, no prob 😃